sklearn/cluster/k_means_.py

```diff
@@ -1046,11 +1039,10 @@ def _mini_batch_step(X, x_squared_norms, centers, counts,
         the distances of each sample to its closest center.
         May not be None when random_reassign is True.
 
-    random_state : int, RandomState instance or None, optional, default: None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default)
+        Determines random number generation for centroid initialization and to
+        pick new clusters amongst observations with uniform probability
```

Comment:
Needs .

Reply:
Do you mean 'Needs' instead of 'Determines'?

Reply:
It needs a `.`

Reply:
Definitely, it needs a '.' . I was too careless about the grammar will keep it in mind. 

Thank you.